### PR TITLE
ECL-376 - Added stubs for financial data

### PR DESCRIPTION
**Pull request by nemanjatomanovic**

### CODE REVIEW

File: app/uk/gov/hmrc/economiccrimelevystubs/controllers/FinancialDetailsController.scala [hunk 1]
```diff
@@ -18,36 +18,36 @@ package uk.gov.hmrc.economiccrimelevystubs.controllers
...
+  readFileService: ReadFileService
 ) extends BackendController(cc) {
 
   def getFinancialDetails(idType: String, idNumber: String, regimeType: String): Action[AnyContent] = Action { _ =>
     idNumber.takeRight(3) match {
-      case "001" | "002"         => Ok(Json.toJson(FinancialDetails(None)))
-      case "003" | "004" | "005" => Ok(Json.toJson(FinancialDetailsStubData.financialDetailsWithPaymentDue))
-      case "400"                 =>
+      case "001" => Ok(readFileService.readFile("FinancialDataClearedObligationResponse"))
+      case "002" => Ok(readFileService.readFile("FinancialDataClearedAndOutstandingObligationResponse"))
```
Comment (TheDanielDoyle) [inline app/uk/gov/hmrc/economiccrimelevystubs/controllers/FinancialDetailsController.scala:35]: Are any of our tests using 004 or 005 for getFinancialDetails?

File: app/uk/gov/hmrc/economiccrimelevystubs/services/ReadFileService.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import play.api.libs.json.{JsValue, Json}
+import javax.inject.Singleton
+import scala.io.Source.fromFile
+
+@Singleton
+class ReadFileService {
+
+  private val BASE_PATH = "resources"
+  def readFile(fileName: String): JsValue = {
```
Comment (TheDanielDoyle) [inline app/uk/gov/hmrc/economiccrimelevystubs/services/ReadFileService.scala:27]: Separate the val and method with a line

Comment (nemanjatomanovic) [reply app/uk/gov/hmrc/economiccrimelevystubs/services/ReadFileService.scala:28]: Fixed.

File: app/uk/gov/hmrc/economiccrimelevystubs/controllers/FinancialDetailsController.scala (hunk 1 above)
Comment (nemanjatomanovic) [reply app/uk/gov/hmrc/economiccrimelevystubs/controllers/FinancialDetailsController.scala]: Test per se not. But this made me thinking, I had to align a bit obligationData with financialData. Also added few more cases, fixed tests and re-structured error responses so they match financialData API spec.

**Review by TheDanielDoyle — APPROVED** [review]
LGTM

**Review by TheDanielDoyle — APPROVED** [review]

**Pull request merged**